src/poetry_plugin_freeze/app.py

```diff
@@ -71,12 +81,13 @@ def get_sha256_digest(content: bytes):
 class IcedPoet:
     factory = Factory()
 
-    def __init__(self, project_dir, wheel_dir="dist"):
+    def __init__(self, project_dir, wheel_dir="dist", exclude_packages=None):
```

Comment:
Nit empty tuple default avoid the if check on usage to go straight into iteration.

Reply:
Oh thanks, after I wrote that check I noticed under the hood list options translate an empty default into `[]` anyway. So can confirm that in a test and get rid of the double check later 👍